Lib/fontTools/feaLib/builder.py

```diff
@@ -1310,30 +1319,38 @@ def build(self):
             self.setLookAheadCoverage_(suffix, st)
             self.setInputCoverage_(input, st)
 
-            st.SubstCount = len([l for l in lookups if l is not None])
+            st.SubstCount = 0
             st.SubstLookupRecord = []
-            for sequenceIndex, l in enumerate(lookups):
-                if l is not None:
-                    if l.lookup_index is None:
-                        raise FeatureLibError('Missing index of the specified '
-                            'lookup, might be a positioning lookup',
-                            self.location)
-                    rec = otTables.SubstLookupRecord()
-                    rec.SequenceIndex = sequenceIndex
-                    rec.LookupListIndex = l.lookup_index
-                    st.SubstLookupRecord.append(rec)
+            for sequenceIndex, lookupList in enumerate(lookups):
```

Comment:
minor: a find the use of `lookupList` (camelCase) and `lookuplist` (lowercase) here and there a bit distracting